changelog.d/9042.misc

```diff
@@ -0,0 +1 @@
+Support routing edu's to multiple instances.
```

Comment:
multiple instances of what?

Reply:
Should we just fold this into the moving sendToDevice changelog?

Reply:
Done that for now.

Otherwise can also change it to:

> Support routing inbound EDU handling to multiple worker instances, depending on  EDU type.

or something